### ISSUE TITLE
[VPAT] White text on red background has poor contrast

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#5142]
Instances where white text ( `#ffffff` ) is rendered on a red background ( `#ff0000` ) have a contrast ratio of 3.99:1.  Per [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum), minimum contrast ratio should be 4.5:1.

This screenshot shows some examples.
![Screen Shot 2020-03-10 at 6 42 44 PM](https://user-images.githubusercontent.com/12703624/76367161-9a6adb00-6302-11ea-8c13-e2db9db7e1ed.png)
